python/sglang/srt/layers/quantization/fp8.py

```diff
@@ -538,7 +561,9 @@ def create_weights(
             layer.register_parameter("w13_weight_scale", w13_weight_scale)
             layer.register_parameter("w2_weight_scale", w2_weight_scale)
 
-            if is_hip_ and get_bool_env_var("CK_MOE"):
+            if (
+                is_hip_
+            ):  # and get_bool_env_var("CK_MOE"): TODO: add check back after triton kernel
```

Comment:
Why remove `get_bool_env_var("CK_MOE")`?

Reply:
We will add triton kernel later - temp remove the check to route all INT4-FP8 calls (in case `CK_MOE` is not given) to AITER kernels. Deliberately by design.